src/cmd_all/src/single_node.rs

```diff
@@ -53,6 +53,16 @@ pub struct SingleNodeOpts {
     node_opts: NodeSpecificOpts,
 }
 
+impl SingleNodeOpts {
+    pub fn new_for_playground() -> Self {
+        let empty_args = vec![] as Vec<String>;
+        let mut opts = SingleNodeOpts::parse_from(empty_args);
+        opts.in_memory = true;
+        opts.max_idle_secs = Some(1800);
```

Comment:
```suggestion
```

...as this was originally for the online playground and it's going to switch to `--in-memory`. Our developers were not fans of this.

Reply:
Sounds good. It's an alias for internal dev then... I will just `hide` it from top-level if possible...

For cloud playground, we can use `./risingwave --in-memory --max-idle-secs=1800` to invoke instead.